tests/lib/util/glob-util.js

```diff
@@ -301,17 +319,19 @@ describe("globUtil", () => {
         it("should not return a file from a glob if it matches a pattern in an ignore file", () => {
```

Comment:
Is this test name still accurate given that we have an `assert.throws` in the test body?